src/vcpkg-utils.ts

```diff
@@ -177,7 +169,25 @@ export class Utils {
   }
 
   public static getAllCachedPaths(baselib: baselib.BaseLib, vcpkgRootDir: string): string[] {
-    let paths = runvcpkglib.getOrdinaryCachedPaths(vcpkgRootDir);
+    const runnerOS = process.env.ImageOS ? process.env.ImageOS : process.platform;
+    const isWindows = runnerOS.slice(0,3) == "win";
+
+    let directorySeparator = '/';
+    if (isWindows) {
+      directorySeparator = '\\';
+    }
+
+    let paths = [ vcpkgRootDir,
+                  "!" + vcpkgRootDir + directorySeparator + "installed",
```

Comment:
Sorry, I got confused because I was mixing legacy mode with manifest mode locally so I had both an `installed` directory in my vcpkg repo and `vcpkg_installed` in the application repo. I think this action should cache both by default so it works regardless of whether vcpkg is in legacy mode or manifest mode.